src/AccessibilityInsights/Modes/StartUpModeControl.xaml.cs

```diff
@@ -1,19 +1,20 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.CommonUxComponents.Dialogs;
+using AccessibilityInsights.SetupLibrary;
```

Comment:
Accessibility.SetupLibrary is the only addition here--the rest have just been sorted